app/controllers/restaurants_controller.rb

```diff
@@ -12,20 +13,24 @@ def create
     end
   end
 
-
   def new
     # @restaurant = Restaurant.find(params[:restaurant_id])
     @restaurant = Restaurant.new
   end
 
-
   def index
     @restaurants = Restaurant.all
   end
 
-
   def show
     @restaurant = Restaurant.find(params[:id])
+    # @restaurant.update(total_wait_time: wait_time)
+    job_id =
+      Rufus::Scheduler.singleton.every '5s' do
+        Rails.logger.info "TEST ME time flies, it's now #{Time.now}"
+        wait_time
```

Comment:
Here every time the "show" action is called, new schedule is going to get create.
Adding more schedule to the existing one.
Maybe we can find a way to set the schedule only once.

Reply:
yes I agree we can use it temporarily until we figure out how to call it once. Perhaps in the application.rb file or something like that when it starts up

Reply:
or perhaps we can stop and start the method when we call it so if its running stop then start again